src/application.ts

```diff
@@ -74,10 +76,8 @@ export default class Application {
           op.data.relationships.hasOwnProperty(relName)
       )
       .reduce((relationAttributes, relName) => {
-        let key =
-          schemaRelationships[relName].foreignKeyName ||
-          `${schemaRelationships[relName].type().type}Id`;
-        let value = (<ResourceRelationshipData>(
+        const key = schemaRelationships[relName].foreignKeyName || `${relName}Id`;
```

Comment:
@spersico In another PR we should set a function to build `${relName}Id`, so can have in a single place the builder for that string.

Reply:
created an issue with this